### ISSUE TITLE
Snippet override

### ISSUE THREAD

**Issue by florian-lefebvre** [novuhq/api-docs#16]
Currently, adding code snippets to the code-examples directory show only them in the api docs. For instance :

- https://docs.novu.co/api/update-subscriber/ has no custom code snippet so all languages are available in the related dropdown
- https://docs.novu.co/api/get-changes/ has a custom code snippet so `Node.js` is the only available snippet

I suggest the custom snippets only act as an override for the language, so that other ones (for instance Go or Python) can remain.
This is critical IMO because changes we make do not wait on the release of the next version of the clients from the monorepo. For instance, #15 added a custom snippet because the related code has been merged in the monorepo but v9 is not released yet (https://github.com/novuhq/novu/pull/1261), so people can't see other way of doing this.

cc @scopsy

**Comment by scopsy** [comment]
@florian-lefebvre agree on this one. 
Will need to make some order in the way we release the docs and code version. right now it's a bit a mess. Will discuss it over discord to find a good structure of managing releases. Any suggestions or places where you saw it work good?

**Comment by florian-lefebvre** [comment]
I think #14 might solve this, at least partially. Maybe we can have a look at how they do it at https://github.com/medusajs/medusa? In docs and packages/medusa-js

**Comment by scopsy** [comment]
Seems like a mix of a `develop` branch and `master` branch, I believe. I think we might head in a similar direction too.

**Comment by florian-lefebvre** [comment]
Yes absolutely, I tend to use this pattern a lot. The master is what goes to production. So for instance, docs should be pushed to the master branch (talking about the monorepo there)
